packages/renderless/src/divider/index.ts

```
@@ -0,0 +1,57 @@
+  }
+
+export const setDividerHeight =
+  ({ state, props, vm, nextTick }: Pick<IDividerRenderlessParams, 'state' | 'props' | 'vm' | 'nextTick'>) =>
+  (): void => {
+    const verticalHeight = '12px'
+
+    if (props.direction === 'vertical') {
+      state.height = props.height || verticalHeight
+    } else {
+      nextTick(() => {
+        const offsetHeight = vm.$refs.text && vm.$refs.text.offsetHeight
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)